src/catalogue/category/catalogueCardView.component.tsx

```diff
@@ -0,0 +1,133 @@
+  onChangeOpenDeleteCategoryDialog: (
+    catalogueCategory: CatalogueCategory
+  ) => void;
+  onChangeOpenEditCategoryDialog: (
+    catalogueCategory: CatalogueCategory
+  ) => void;
+  onChangeOpenSaveAsDialog: (catalogueCategory: CatalogueCategory) => void;
+  handleToggleSelect: (catalogueCategory: CatalogueCategory) => void;
+  selectedCategories: CatalogueCategory[];
+}
+
+function CardView(props: CardViewProps) {
+  const {
+    catalogueCategoryData,
+    onChangeOpenDeleteCategoryDialog,
+    onChangeOpenEditCategoryDialog,
+    onChangeOpenSaveAsDialog,
+    handleToggleSelect,
+    selectedCategories,
+  } = props;
+
+  const [page, setPage] = React.useState(1);
+  const [paginationResults, setPaginationResults] = React.useState<number>(30);
+  const startIndex = (page - 1) * paginationResults;
+  const endIndex = startIndex + paginationResults;
+  const displayedCatalogueCategories = catalogueCategoryData?.slice(
+    startIndex,
+    endIndex
+  );
+
+  return (
+    <Grid container flexDirection={'column'}>
+      <Grid container item xs={12} maxHeight={'675px'} overflow={'auto'}>
+        {displayedCatalogueCategories?.map((item, index) => (
+          <Grid
+            item
+            key={index}
+            xs={12}
+            sm={6}
+            md={4}
+            flexDirection={'column'}
+            alignContent={'center'}
+          >
+            <CatalogueCard
+              {...item}
+              onChangeOpenDeleteDialog={onChangeOpenDeleteCategoryDialog}
+              onChangeOpenEditDialog={onChangeOpenEditCategoryDialog}
+              onChangeOpenSaveAsDialog={onChangeOpenSaveAsDialog}
+              onToggleSelect={handleToggleSelect}
+              isSelected={selectedCategories.some(
+                (selectedCategory: CatalogueCategory) =>
+                  selectedCategory.id === item.id
+              )}
+            />
+          </Grid>
+        ))}
+      </Grid>
+      <Grid
+        container
+        item
+        alignItems="center"
+        justifyContent="right"
+        xs={12}
+        padding={2}
+        position={'fixed'}
+        bottom={12}
+        right={0}
+      >
+        <Grid item>
+          <FormControl variant="standard" sx={{ margin: 1, minWidth: '120px' }}>
+            <InputLabel>{'Max Results'}</InputLabel>
+            <Select
+              value={paginationResults}
+              inputProps={{
+                name: 'Max Results',
+                id: 'select-max-results',
+              }}
+              onChange={(event) => {
+                setPaginationResults(+event.target.value);
+                setPage(1);
+              }}
+              label={'Max Results'}
+            >
+              <MenuItem value={'30'}>30</MenuItem>
+              <MenuItem value={'45'}>45</MenuItem>
+              <MenuItem value={'60'}>60</MenuItem>
+            </Select>
+          </FormControl>
+        </Grid>
+
+        <Grid item>
+          <Pagination
```

Comment:
I've tried to implement it, but as it is meant to be used in the footer of a table it doesn't look like how we have it in the table views. And it also doesn't work due to the fact it requires 'rows' to be able to render, but we are using `cards` not in a row format

Reply:
Ah yes I see now, I forgot it uses MRT_TablePagination, which actually uses Pagination manually like you have done instead of TablePagination. Instead of that could you use `disableUnderline` and move text to the left, then it closer to it at least.